### DISCUSSION TITLE
Linked records title bug

### DISCUSSION THREAD

**Discussion by yobotn** [nocodb/nocodb#726] [Q&A] [answered]
instead of showing the linked record value, it shows its time.

fresh install 0.80.0

droplet Basic / 4 GB / 2 vCPUs

![image](https://user-images.githubusercontent.com/40774550/139589855-fd6d80a4-6228-4e98-bcab-7e5a71e089bf.png)

**Comment by o1lab** [comment, marked as answer]
If you go to categories and hover on a column other than date. You should be able to set that as primary value. And then linked record value can be seen.
